Task09/script.js

```diff
@@ -0,0 +1,35 @@
+(() => {
+  'use strict';
+
+	const avoidDuplication = (event) => {
+		const apiURL = kintone.api.url('/k/v1/records.json'); //https://…　ではなく、kintone.api.url() を使うことで短縮できる
+		//新規レコード作成をデフォルトとしてparameterの変数を定義
+		const paramsField = (event.type === 'app.record.create.submit') ? ['重複禁止項目'] : ['重複禁止項目', '$id'];
+		const paramsQuery = (event.type === 'app.record.create.submit') ? `重複禁止項目 = "${event.record.重複禁止項目.value}"` : `重複禁止項目 = "${event.record.重複禁止項目.value}" and $id not in ("${event.record.$id.value}")`;
+		const getParams = {
+			app: kintone.app.getId(),
+			fields: paramsField,
+			paramsQuery,
```

Comment:
キーを追加してください！
（おそらく現状のコードだと重複していなくてもアラートが出ちゃうはず）

```suggestion
			query: paramsQuery,
```